newIDE/app/src/Leaderboard/LeaderboardProvider.js

```diff
@@ -202,11 +205,11 @@ const LeaderboardProvider = ({ gameId, children }: Props) => {
   });
 
   const listLeaderboards = React.useCallback(
-    async () => {
+    async (silent?: boolean) => {
```

Comment:
What does "silent" do ? I did not understand why we need to set an interval and fetch leaderboards when we are resetting.

Reply:
It is ill-named. Without it, when the leaderboards are fetched, when we are waiting for the response, an error is displayed on the UI. So, with it, it's more a `background fetch`